Author: Ana Beatriz Guerrero Lopez <ana@debian.org>
Author: Pino Toscano <pino@debian.org>
Description: add a few Debian feeds to akregator
 Include Planet Debian and Debian News as default in akregator
Forwarded: not-needed
Origin: vendor
Last-Update: 2012-04-17

--- a/akregator/src/akregator_part.cpp
+++ b/akregator/src/akregator_part.cpp
@@ -133,6 +133,20 @@ namespace {
         look.setAttribute("xmlUrl","http://www.kde.org/kde-look-content.rdf");
         mainFolder.appendChild(look);
 
+        QDomElement debianFolder = doc.createElement( "outline" );
+        debianFolder.setAttribute("text","Debian");
+        body.appendChild(debianFolder);
+
+        QDomElement pland = doc.createElement( "outline" );
+        pland.setAttribute("text",i18n("Planet Debian"));
+        pland.setAttribute("xmlUrl","http://planet.debian.org/rss20.xml");
+        debianFolder.appendChild(pland);
+
+        QDomElement dtimes = doc.createElement( "outline" );
+        dtimes.setAttribute("text",i18n("Debian News"));
+        dtimes.setAttribute("xmlUrl","http://www.debian.org/News/news");
+        debianFolder.appendChild(dtimes);
+
         return doc;
     }
 }
